src/styles/blocks/about.scss

```diff
@@ -0,0 +1,69 @@
+.about {
+  &__info {
+    &--title {
+      margin-bottom: 10px;
+      @extend %text-h2;
+    }
+
+    &--paragraph {
+      margin-bottom: 30px;
+    }
+
+    &--desktop-paragraph {
+      display: none;
+
+      @include Desktop {
+        display: block;
+        margin-bottom: 40px;
+      }
+    }
+
+    &--link {
+      @extend %links;
+
+      &:hover,
+      &:active {
+        padding-bottom: 10px;
+        border-bottom: 1px solid $secondary-color;
+      }
+    }
+  }
+
+  &__main-image {
+    display: none;
+    background-image: url(../images/png/about_tablet.png);
+    @extend %bacground-images-settings;
```

Comment:
There is a typo in the class name: %bacground-images-settings should be %background-images-settings.